services/src/datasets/storage.rs

```diff
@@ -99,6 +104,17 @@ fn validate_main_file(main_file: &String) -> Result<(), ValidationError> {
     Ok(())
 }
 
+fn validate_tags(tags: &Vec<String>) -> Result<(), ValidationError> {
+    for tag in tags {
+        if tag.is_empty() || tag.contains('/') || tag.contains("..") || tag.contains(' ') {
```

Comment:
should the "uploads" tag be reserved? should upper/lower case play a role?

Reply:
good question. currently i check if upload is set and then add it if not (only for uploads).